src/components/shared/UserInfoPopover/UserReputation.tsx

```diff
@@ -16,7 +15,7 @@ const displayName = `UserInfoPopover.UserReputation`;
 
 interface Props {
   colony: Colony;
-  userReputationForTopDomains: UserDomainReputation[];
+  userReputationForTopDomains: any[];
```

Comment:
This gives me a cold shudder. Can't we type it?

Reply:
It's the old world code, I'd rather just delete the entire file but I wanted to keep this PR concise

Reply:
Ah I see. Let's move on then :)